docs/source/quickstart.rst

```diff
@@ -3,76 +3,169 @@
 Quickstart
 ==========
 
-Download wis2box and start using Malawi test data:
+Requirements and dependencies
+-----------------------------
+
+wis2box requires the following prior to installation:
+
+.. csv-table::
+   :header: Requirement,Version
+   :align: left
+
+   `Python`_,3.8 (or greater)
+   `Docker Engine`_, 20.10.14 (or greater)
+   `Docker Compose`_1.29.2 (or greater)
+
+If these are already installed, you can skip to installing wis2box.
+
+- To install Python, follow `Python installation`_.
+- To install Docker, follow `Docker Engine installation`_.
+- To install Docker Compose, follow `Compose installation`_.
+
+Successful installation can be confirmed by inspecting the versions on your system.
+
+.. code-block:: bash
+
+    docker version
+    docker-compose version
+    python3 -V
 
 .. code-block:: bash
 
     git clone https://github.com/wmo-im/wis2box.git
     cd wis2box
 
 
-For the purposes of a quickstart, this deployment expects the test environment, which includes data and metadata, and runs on localhost. This
-is done by using the test environment file:
+wis2box configuration
+---------------------
+
+Wis2box will read environment variables from dev.env. A baseline is provided in dev.env.example. Copy dev.env.example to dev.env
```

Comment:
This is actually wrong, the majority of the environment variables are defined in the docker/default.env file. This includes specifying the location of the mapping file.

Reply:
the docker-compose will overwrite values defined docker/default.env with those defined in dev.env. So the user should edit them in dev.env.